app/http.php

```diff
@@ -91,19 +91,12 @@
         /** @var array $collections */
         $collections = Config::getParam('collections', []);
 
-        if (!$dbForConsole->exists(App::getEnv('_APP_DB_SCHEMA', 'appwrite'))) {
+        try {
             $redis->flushAll();
-
             Console::success('[Setup] - Creating database: appwrite...');
-
             $dbForConsole->create(App::getEnv('_APP_DB_SCHEMA', 'appwrite'));
-        }
-
-        try {
-            Console::success('[Setup] - Creating metadata table: appwrite...');
-            $dbForConsole->createMetadata();
-        } catch (\Throwable $th) {
-            Console::success('[Setup] - Skip: metadata table already exists');
+        } catch (\Exception $e) {
+            Console::error('Failed to create database: ' . $e->getMessage());
```

Comment:
```suggestion
            Console::success('[Setup] - Skip: metadata table already exists');
```
Same as previous. I'll make sure we improve this and have proper method to check initialization of the DB.